src/components/category-form/category-form.js

```diff
@@ -82,7 +81,7 @@ const CategoryForm = ({ category, id, edit }) => {
   } = useFormik({
     validationSchema: categoryValidationSchema,
     initialValues: {
-      categoryImage: IMG_URL + category.images.thumbnail || '',
+      categoryImage: category.images.thumbnail || '',
```

Comment:
Why you remove possibility to see photo when we edit category?

Reply:
fixed